### PR TITLE
Add all valid regions as of July 19th 2018...

### PR DESCRIPTION
**Pull request by mgajda**
Just to make it work for all non-US people out of the box...

### CODE REVIEW

**Comment by mgajda** [comment]
Just update of #61, #55, and #54.

File: cmd/root.go [hunk 1]
```diff
@@ -105,13 +105,16 @@ CloudWatch Logs, and Amazon Route 53 into an easy-to-use CLI.`,
...
+                var foundRegion bool = false
 		for _, validRegion := range validRegions {
 			if region == validRegion {
+                                foundRegion = true
 				break
 			}
-
-			console.IssueExit("Invalid region: %s [valid regions: %s]", region, strings.Join(validRegions, ", "))
 		}
+		if (! foundRegion) {
```
Comment (jpignata) [inline cmd/root.go]: Can you run this through gofmt?

File: docker/main.go [hunk 2]
```diff
@@ -46,7 +46,7 @@ func (repository *Repository) Login(username, password string) {
 
 func (repository *Repository) Build(tag string) {
 	console.Debug("Building Docker image [%s]", repository.UriFor(tag))
-	console.Shell("docker build --tag %s .", repository.UriFor(tag))
+	console.Shell("docker build --rm=false --tag %s .", repository.UriFor(tag))
```
Comment (jpignata) [inline docker/main.go]: Hmm - this seems unrelated to the fix proposed. Happy to merge it separately with some explanation about the benefits.

**Comment by tisba** [comment]
Please keep in mind that there seems to be more changes required, as pointed out in https://github.com/jpignata/fargate/pull/54 (issues regarding security groups).

**Comment by jpignata** [comment]
Thanks, @tisba. I suppose we can handle that separately. It seems possible to me off the top of my head that default VPCs have some kind of behavior difference between US East (N. Virginia) and elsewhere, though I don't know of any off the top of my head.

**Comment by tisba** [comment]
I don't recall exactly what the issue was. The only solution I found back then was to provide an explicit security group.

¯\_(ツ)_/¯

File: cmd/root.go (hunk 1 above)
Comment (mgajda) [reply cmd/root.go]: Sure, will.

File: docker/main.go (hunk 2 above)
Comment (mgajda) [reply docker/main.go]: Yes, I will put it into separate branch. Shortens build time when you frequently rebuild the same image.

**Comment by mgajda** [comment]
Hi John,
I implemented your suggestions:
* squashing it into a single commit
* running through `gofmt`
* putting `docker` into separate patch (see #63)

**Comment by tisba** [comment]
Are you planning to make a new release, @jpignata?

**Comment by pahud** [comment]
please release 0.2.4 with this PR, thanks.
